Dockerfile-csi-controller

```diff
@@ -29,7 +29,7 @@ RUN ./csi_pb2.sh
 RUN pip3 install .
 
 
-FROM registry.access.redhat.com/ubi8/python-38:1-77
+FROM registry.access.redhat.com/ubi8/python-38:1-80
```

Comment:
what about the `FROM` [above](https://github.com/IBM/ibm-block-csi-driver/blob/16ad98f5c311925648a2bcb3464b8db7cc0c8b33/Dockerfile-csi-controller#L15)?

Reply:
same in .test

Reply:
not relevant since https://github.com/IBM/ibm-block-csi-driver/pull/438

Reply:
it looks there that both images in the file have been downgraded, why is the second one is ok to bump?

Reply:
because it doesn't fail on compilation, it just contains the complied files